js/package.json

```diff
@@ -1,17 +1,17 @@
 {
   "name": "google-protobuf",
-  "version": "3.0.0-alpha.6",
+  "version": "3.0.0-alpha.6.2",
   "description": "Protocol Buffers for JavaScript",
   "main": "google-protobuf.js",
-  "dependencies": {
+  "files": "google",
```

Comment:
"files" is supposed to be an array? https://docs.npmjs.com/files/package.json#files